webrender/src/renderer.rs

```diff
@@ -552,6 +553,11 @@ impl Renderer {
         let data64_texture = VertexDataTexture::new(&mut device);
         let data128_texture = VertexDataTexture::new(&mut device);
 
+        let dummy_mask_texture = device.create_texture_ids(1)[0];
```

Comment:
I wonder if this will cause unnecessary batch breaks when there are clips that don't have masks. We could probably insert a dummy mask into the atlas instead of creating a separate texture. But that doesn't matter for now, we can do it as a follow up if it's an issue.